From f10483ee2abf8c9e36746a8c139d3296420e2982 Mon Sep 17 00:00:00 2001
From: Ray Chang <ray.chang@technexion.com>
Date: Fri, 22 May 2026 13:11:53 +0800
Subject: [PATCH 1/3] imx8m: declare dram timing info for 4GB single-rank DDR

Introduce the extern declaration for 'dram_timing_4gb_1rank' in the i.MX8M
DDR architecture header.
---
 arch/arm/include/asm/arch-imx8m/ddr.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/include/asm/arch-imx8m/ddr.h b/arch/arm/include/asm/arch-imx8m/ddr.h
index 979fd39326b..fe5773bb6f8 100644
--- a/arch/arm/include/asm/arch-imx8m/ddr.h
+++ b/arch/arm/include/asm/arch-imx8m/ddr.h
@@ -707,6 +707,7 @@ extern struct dram_timing_info dram_timing_1gb;
 extern struct dram_timing_info dram_timing_2gb;
 extern struct dram_timing_info dram_timing_3gb;
 extern struct dram_timing_info dram_timing_4gb;
+extern struct dram_timing_info dram_timing_4gb_1rank;
 extern struct dram_timing_info dram_timing_6gb;
 extern struct dram_timing_info dram_timing_8gb;
 
-- 
2.39.5

